src/importexport/mei/internal/meireader.cpp

```diff
@@ -0,0 +1,71 @@
+/*
+ * SPDX-License-Identifier: GPL-3.0-only
+ * MuseScore-CLA-applies
+ *
+ * MuseScore
+ * Music Composition & Notation
+ *
+ * Copyright (C) 2021 MuseScore BVBA and others
+ *
+ * This program is free software: you can redistribute it and/or modify
+ * it under the terms of the GNU General Public License version 3 as
+ * published by the Free Software Foundation.
+ *
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ * GNU General Public License for more details.
+ *
+ * You should have received a copy of the GNU General Public License
+ * along with this program.  If not, see <https://www.gnu.org/licenses/>.
+ */
+
+#include <QFileInfo>
+#include <QMessageBox>
+
+#include "meireader.h"
+
+#include "meiimporter.h"
+
+#include "libmscore/masterscore.h"
+#include "engraving/engravingerrors.h"
+
+using namespace mu::iex::mei;
+using namespace mu::engraving;
+
+mu::Ret MeiReader::read(MasterScore* score, const io::path_t& path, const Options& options)
+{
+    if (!QFileInfo::exists(path.toQString())) {
```

Comment:
We prefer not using Qt dependencies in new code where we don't need to (which basically means: nowhere except in UI-related code). That means for example: 
- using `IFileSystem` (via `INJECT`) instead of `QFileInfo`
- `mu::String` instead of `QString`
- `mu::io::File` instead of `QFile`

These classes are designed specifically to be an easy replacement of the Qt APIs (and some are under the hood still using Qt APIs in their backend, but that allows us to create different backends too).